### ISSUE TITLE
Is it normal to stay at this step for a long time?

### ISSUE THREAD

**Issue by 1023011930** [EMBL-PKU/BASALT#36]
![image](https://github.com/EMBL-PKU/BASALT/assets/89200805/9546bb42-dedb-4ea6-8ef9-73dc1f6a97b6)

![image](https://github.com/EMBL-PKU/BASALT/assets/89200805/de813fd6-b50f-4a8d-9092-fca58224478a)

**Comment by 1023011930** [comment]
I would appreciate a reply from you guys!

**Comment by 1023011930** [comment]
[BASALT_log.txt](https://github.com/EMBL-PKU/BASALT/files/15494904/BASALT_log.txt)
[Basalt_checkpoint.txt](https://github.com/EMBL-PKU/BASALT/files/15494908/Basalt_checkpoint.txt)

**Comment by EMBL-PKU** [comment]
Hi, I am sorry for the slow reply. Please refer to the new version of BASALT V 1.0.2. I think we just corrected this bug and I just uploaded it to github. Please let me know if this happened again. Thank you very much for your patient.

**Comment by 1023011930** [comment]
If I want to update it, can I just replace the old zip file with the .zip file from github?

**Comment by EMBL-PKU** [comment]
Yes, please just replace those script under your conda BASALT folder. I just uploaded a new version 1.2.1 into github. Please use the newest version. Thank you!

**Comment by 1023011930** [comment]
> Yes, please just replace those script under your conda BASALT folder. I just uploaded a new version 1.2.1 into github. Please use the newest version. Thank you!
You're so efficient! It scares me.
